src/hyperloglog.c

```diff
@@ -1857,7 +1857,7 @@ void pfselftestCommand(client *c) {
      * structure are accessible and that setting their values both result in
      * the correct value to be retained and not affect adjacent values. */
     for (j = 0; j < HLL_TEST_CYCLES; j++) {
-        /* Set the HLL counters and an array of unsigned byes of the
+        /* Set the HLL counters and an array of unsigned bytes of the
```

Comment:
this is amusing and was an in-the-neighborhood catch+fix (the word is in the dictionary, it just isn't the right word). I've since added a [rule to spell-check-this](https://github.com/check-spelling/spell-check-this/commit/b47fb37a4e11aef564092cdcde60ab345549207b) because of this.